src/app/capture_thread.cpp

```diff
@@ -125,6 +129,8 @@ void CaptureThread::selectCaptureMethod() {
     new_capture = captureV4L;
   } else if(captureModule->getString() == "DC 1394") {
     new_capture = captureDC1394;
+  } else if(captureModule->getString() == "Flycapture") {
```

Comment:
Needs to be wrapped in #ifdef , please see https://github.com/RoboCup-SSL/ssl-vision/blob/ecd668a8d5f1afc416f05aad34be4cb331e9244f/src/app/capture_thread.cpp#L120